### PR TITLE
Initial pre-summit script…

### PR DESCRIPTION
**Pull request by vdemeester**
… partly imported from `tektoncd/pipeline`

/hold

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

### CODE REVIEW

File: test/presubmit-tests.sh [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+# This script runs the presubmit tests; it is started by prow for each PR.
+# For convenience, it can also be executed manually.
+# Running the script without parameters, or with the --all-tests
+# flag, causes all tests to be executed, in the right order.
+# Use the flags --build-tests, --unit-tests and --integration-tests
+# to run a specific set of tests.
+
+# Custom configuration of presubmit tests
+readonly DISABLE_MD_LINTING=${DISABLE_MD_LINTING:-0}
```
Comment (matzew) [inline test/presubmit-tests.sh]: lint is ... great! :rofl:

**Comment by vdemeester** [comment]
Race condition issue : https://github.com/tektoncd/dashboard/issues/73

**Comment by vdemeester** [comment]
/assign @abayer 
/assign @skaegi 

for `/lgtm` and `/approve`

**Comment by vdemeester** [comment]
oups sorry @abayer =)

/assign @a-roberts

**Comment by a-roberts** [comment]
Output from this looks great and any improvements can be made after - thank you so much @vdemeester for working on this!

/approve

**Comment by a-roberts** [comment]
/lgtm

(poking Prow)

**Pull request merged**